invenio/legacy/refextract/api.py

```diff
@@ -17,82 +17,95 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-"""This is where all the public API calls are accessible
+"""This is where all the public API calls are accessible.
 
 This is the only file containing public calls and everything that is
 present here can be considered private by the invenio modules.
 """
 
 
-import os
-import sys
+from invenio.base.config import CFG_TMPSHAREDDIR
```

Comment:
You need to use either `from invenio.base.globals import cfg` or `from invenio.config import CFG_TMPSHAREDDIR`.

--
PS: Please re-order commits according to PEP8. Thank you!